packages/widget/src/pages/SettingsPage/ResetSettingsButton.tsx

```diff
@@ -35,25 +44,43 @@ export const ResetSettingsButton: React.FC = () => {
     toggleDialog();
   };
 
+  const isCustomRouteSettings =
+    tools?.bridges?.length !== enabledBridges?.length ||
+    tools?.exchanges?.length !== enabledExchanges?.length;
+
```

Comment:
We should take into account all settings that can affect route availability, including route priority, gas price, slippage 🙂

Reply:
got it, makes sense. I'll include them

Reply:
I'm unable to find the type for `gasPrice` in `WidgetConfig` interface, am I missing something here?

Reply:
Yes, there is no option for `gasPrice` in `WidgetConfig`, but `gasPrice` can be modified by the user, so we also need to consider that.

Reply:
Done, pushed the changes